ci-operator/step-registry/ipi/install/loki/ipi-install-loki-commands.sh

```diff
@@ -75,6 +77,7 @@ spec:
   - persistentVolumeClaim
   - secret
 EOF
+fi
```

Comment:
nit: you can extend this all the way through the next RBAC as @bparees pointed out in the other PR:

Reply:
Sorry, missed that.